### PR TITLE
Flip default in checkAuthorized

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: lib/Controller/ApiController.php [hunk 1]
```diff
@@ -129,11 +129,11 @@ private function checkAuthorized() {
...
 		}
 
-		if (!$this->groupManager->isAdmin($user->getUID())) {
-			return false;
+		if ($this->groupManager->isAdmin($user->getUID())) {
+			return true;
 		};
 
-		return true;
+		return false;
```
Comment (blizzz) [inline lib/Controller/ApiController.php:136]: What about?

``return $this->groupManager->isAdmin($user->getUID());``

It returns the desired boolean already.

Comment (J0WI) [reply lib/Controller/ApiController.php:136]: Make sense, but I'd prefer to merge https://github.com/nextcloud/serverinfo/pull/288/files#diff-53e5a55431f07eddf1b35d8e80157e02bb1955bda3e3fcaf20bb6dc8c1f88e56 first then.

Comment (blizzz) [reply lib/Controller/ApiController.php:136]: thanks! reviewed it, also needs a rebase.

**Review by blizzz — APPROVED** [review]

**Comment by blizzz** [comment]
conflicts :face_with_head_bandage:

**Comment by J0WI** [comment]
Rebased.

**Comment by blizzz** [comment]
Thanks :)

**Pull request merged**